add two more methods to the monotonic section (#81)
ietf-wg-uuidrev/rfc4122bis: draft-ietf-uuidrev-rfc4122bis.md

{{distributed_shared_knowledge}}.

Implementations SHOULD choose one method for single-node UUID implementations
that require batch UUID creation, or are otherwise concerned about monotonicity

> with high frequency UUID generation.

Author: This is the same edit from here https://github.com/ietf-wg-uuidrev/rfc4122bis/pull/79#discussion_r1161177270

  SHOULD NOT be used by implementations that favor unguessiblity, as the resulting
  values are easily guessable.

Re-randomize Until Monotonic (Method 3):
: If the performance impact is acceptable (depending on the UUID version,
  granularity of the system clock, and expected throughput of values
  generated per clock value), an implementation can, in the case
  of multiple UUID values using the same clock tick, simply regenerate
  a new random value as many times as needed in order to produce a UUID that is
  ordered after the last one.  The primary benefit of this approach is simplicity

>   of implementation. The down sides to this approach include the fact that

Reviewer: ```suggestion
  of implementation. One downside to this approach is the fact that
```

>   generation will become increasing slow for a given timestamp value as

Reviewer: ```suggestion
  generation will become increasingly slow for a given timestamp value as
```

>   more attempts are required to product monotonic output, and the fact that

Reviewer: ```suggestion
  more attempts are required to produce monotonic output, and the fact that
```
